### PR TITLE
Features one

### PR DESCRIPTION
**Pull request by asnganje**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (Ugboaja-Uchechi) [inline README.md]: - [ ] Please remove all sections that aren't required so that your README looks more professional.

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @asnganje , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Great job setting up your repo 👍 
- Your project is following the GitHub Flow rules.

## Required Changes ♻️

- [ ] Please make sure you add all the linters Microverse suggested for this project. Here is a [list of instructions](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to go about it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It would be best to provide descriptions of your project in the pull request so it looks professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (asnganje) [reply README.md]: @Ugboaja-Uchechi Thank you for the review. Can you help me with how I can add the changes

Comment (Ugboaja-Uchechi) [reply README.md]: Hi @asnganje, sure. You can contact me on slack @tired coder so we can go through it together.

**Review by Reem-lab — APPROVED** [review]
## Hi @asnganje , 👋🏻👋🏻 

Your project is complete! 🎉🎉🔥🔥
 There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
 - Linters checked passes ✅✅
 - Followed GitHub flow ✅✅

## Optional suggestions
[OPTIONAL] Kindly add description to you summary and your pull request to summarize the changes you did in this specific milestone 👌🏻💪🏻🚀

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**